adapters/Unity/UnityAdapter/GADMAdapterUnityProtocol.h

```diff
@@ -24,4 +24,7 @@
 /// Returns placement ID for either reward-based video ad or interstitial ad of Unity Ads network.
 - (NSString *)getPlacementID;
 
+/// Called when an ad fails to load.
+- (void)didFailToLoadWithError:(nonnull NSError *)error;
```

Comment:
Possible this function without implementation failed to be caught by the linker because of weak link.
When errors happen and call [delegete didFailToLoadWithError: error], it will cause signal SIGABRT.

`GADMAdapterUnity` and `GADMAdapterUnityRewardedAd` would need to implement this method.
Test cases validating the failure path would help preventing this kind of run time error too.

Reply:
Acknowledged. @chlandacastro these two classes need to implement the new method since it's required and those other classes conform to this protocol.